site/src/components/pagination.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Pagination
+description: Pagination related components in the Atlas Design System
+template: standard
+---
+
+# Pagination
+
+The pagination component consists of several elements:
+
+- `pagination-previous` and `pagination-next` for incremental navigation
+- `pagination-list` which displays page items for navigation to individual pages
+- `pagination-link` for page numbers and `pagination-ellipsis` for range numbers
+
+All elements are optional so you can compose your pagination as you wish.
+
+```html
+<nav class="pagination" role="navigation" aria-label="pagination">
+	<a class="pagination-previous" aria-label="previous">
+		<span class="icon" aria-hidden="true">
+			<span class="docon docon-arrow-left"></span>
+		</span>
+	</a>
+	<a class="pagination-next" aria-label="next">
+		<span class="icon" aria-hidden="true">
+			<span class="docon docon-arrow-right"> </span>
+		</span>
+	</a>
+	<ul class="pagination-list">
+		<li>
+			<a class="pagination-link" aria-label="Goto page 1">1</a>
```

Comment:
nit: When I navigate to the anchors with a screen reader, it reads what sounds like "go do page 1". It sounds more like "go to" if you add a space between "go" and "to".